test/defs/test_defs_us.rb

```diff
@@ -19,6 +19,10 @@ def test_us
  Date.civil(2008,12,25) => 'Christmas Day'}.each do |date, name|
   assert_equal name, (Holidays.on(date, :us)[0] || {})[:name]
 end
+{Date.civil(2008, 3, 31) => 'Cesar Chavez Day',
+ Date.civil(2008, 11, 28) => 'Day after Thanksgiving'}.each do |date, name|
```

Comment:
Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.